Formula/mingw-w64.rb

```diff
@@ -0,0 +1,177 @@
+class MingwW64 < Formula
+  desc "Minimalist GNU for Windows and GCC cross-compilers"
+  homepage "https://mingw-w64.org/"
+  url "https://downloads.sourceforge.net/project/mingw-w64/mingw-w64/mingw-w64-release/mingw-w64-v5.0.1.tar.bz2"
+  sha256 "9bb5cd7df78817377841a63555e73596dc0af4acbb71b09bd48de7cf24aeadd2"
+
+  # Multilib does not work currently
+  # option "without-multilib", "Build 64-bit compiler with multilib support"
+
+  depends_on "gmp"
+  depends_on "mpfr"
+  depends_on "libmpc"
+  depends_on "isl"
+
+  # Apple's makeinfo is old and has bugs
+  depends_on "texinfo" => :build
+
+  resource "binutils" do
+    url "https://ftpmirror.gnu.org/binutils/binutils-2.27.tar.gz"
+    sha256 "26253bf0f360ceeba1d9ab6965c57c6a48a01a8343382130d1ed47c468a3094f"
+  end
+
+  resource "gcc" do
+    url "https://ftpmirror.gnu.org/gcc/gcc-6.3.0/gcc-6.3.0.tar.bz2"
+    sha256 "f06ae7f3f790fbf0f018f6d40e844451e6bc3b7bc96e128e63b09825c1f8b29f"
+  end
+
+  def target_archs
+    ["i686", "x86_64"]
+  end
+
+  def install
+    bin.mkdir
```

Comment:
Well, apparently not such as no-op… `Errno::EEXIST: File exists - /usr/local/Cellar/mingw-w64/5.0.1/bin`

I am retesting with `mkdir_p` which should fix that.